src/etl/sds/executor/input/input.py

```diff
@@ -0,0 +1,64 @@
+import boto3
+from etl_utils.trigger.logger import log_action
+from etl_utils.trigger.notify import notify
+from event.environment import BaseEnvironment
+from event.step_chain import StepChain
+
+from .steps import _process_sqs_message, steps
+
+
+class ChangelogTriggerEnvironment(BaseEnvironment):
+    STATE_MACHINE_ARN: str
+    NOTIFY_LAMBDA_ARN: str
+    TRUSTSTORE_BUCKET: str
+    CPM_FQDN: str
+    LDAP_HOST: str
+    ETL_BUCKET: str
+    LDAP_CHANGELOG_USER: str
+    LDAP_CHANGELOG_PASSWORD: str
+
+
+S3_CLIENT = boto3.client("s3")
+STEP_FUNCTIONS_CLIENT = boto3.client("stepfunctions")
+LAMBDA_CLIENT = boto3.client("lambda")
+ENVIRONMENT = ChangelogTriggerEnvironment.build()
+
+
+CACHE = {
+    "s3_client": S3_CLIENT,
+    "step_functions_client": STEP_FUNCTIONS_CLIENT,
+    "state_machine_arn": ENVIRONMENT.STATE_MACHINE_ARN,
+    "etl_bucket": ENVIRONMENT.ETL_BUCKET,
+}
+
+
+def handler(event=dict, context=None):
+    for message in event["Records"]:
+        process_message(message)
+
+    return None  # Message removed from sqs queue
```

Comment:
I'd add a more verbose comment here to warn not to change this return value as it's the expected return from a successful SQS lambda